.ci/setup_env_sles.sh

```diff
@@ -27,7 +27,8 @@ echo "Install perl-IPC-Run"
 sudo -E zypper -n install perl-IPC-Run
 
 echo "Add repo for moreutils"
-moreutils_repo="https://download.opensuse.org/repositories/utilities/SLE_${VERSION//-/_}/utilities.repo"
+# For SLES 15 SP2 no utilities repository available
+moreutils_repo="https://download.opensuse.org/repositories/utilities/SLE_15_SP1/utilities.repo"
```

Comment:
@GabyCT - do we know why the utilities repo isn't available? fwics SP2 came out 3 months ago. Since an SP is effectively a "point release" and since SLES is an LTS, this looks a bit worrying to me.

@rhafer - can you comment on this please?

Reply:
@jodh-intel no I don't know why is not available

Reply:
@jodh-intel @GabyCT  Hm, are you sure that the repository is really still needed? AFAICS the "moreutils" package is also available via the `PackageHub` module meanwhile, which is added a few lines below:

https://github.com/kata-containers/tests/blob/cd99b4cb7eeb2db87e09d9e63e0d672c18745063/.ci/setup_env_sles.sh#L41

Would you mind trying without adding the `utilities` repository?